assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -58,13 +58,35 @@ const RangeInputWrapper = styled.div`
   margin: 0 auto 75px auto;
 `;
 
-const Rectangle = styled.div`
+const FlexGrowRangeInput = styled(RangeInput)`
+  flex-grow: 1;
+`;
+
+const Rectangle = styled.button.attrs(({ disabled }) => ({
```

Comment:
This should be unnecessary. Just `styled.button` without the `attrs` part works, since `disabled` is a default attribute of a button.

Reply:
Okay

Reply:
Done